### PR TITLE
fix various delegate docs

### PR DESCRIPTION
**Pull request by raghuAtWings**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Comment by ravilach** [comment]
@raghuAtWings where you able to test these changes locally? e.g run a `yarn build` and then `yarn serve` to check them. Looks like build has errors, not sure if has to do with some refactoring activity going on CC @kat-enos

File: docs/platform/2_Delegates/advanced-installation/automate-delegate-installation.md [hunk 1]
```diff
@@ -65,18 +65,22 @@ The Delegate Environment Variables are described in the relevant Delegate instal
 
 ### Step 2: Rename the New Delegate
 
-The process you use to rename a delegate depends on its type. For Docker delegates, you change the name in one environment variable in the Docker compose file. For the Kubernetes delegate, you change multiple instances of the name.
+The process you use to rename a delegate depends on its type. For Docker delegates, you change the name in the command line. For the Kubernetes delegate, you change multiple instances of the name.
```
Comment (kat-enos) [inline docs/platform/2_Delegates/advanced-installation/automate-delegate-installation.md:68]:
```suggestion
The process you use to rename a delegate depends on its type. For a Docker delegate, you change the name on the command line. For a Kubernetes delegate, you change multiple instances of the name.
```

File: docs/platform/2_Delegates/customize-delegates/build-custom-delegate-images-with-third-party-tools.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ You can build on either of the following Harness-provided images.
 | Harness Delegate Docker image | A publicly available Docker image providing Harness Delegate. |
 | Harness Minimal Delegate Docker image | A minimal delegate image available in Docker Hub at <https://hub.docker.com/r/harness/delegate/tags>. |
 
-You can use the `latest` version minimal image from the Docker repository.
+You can use the  last published `YY.MM.XXXXX` version minimal image from the Docker repository.
```
Comment (kat-enos) [inline docs/platform/2_Delegates/customize-delegates/build-custom-delegate-images-with-third-party-tools.md:34]:
```suggestion
You can use the  last published `YY.MM.XXXXX` version of the minimal image from the Docker repository.
```

File: docs/platform/2_Delegates/delegate-reference/common-delegate-profile-scripts.md [hunk 3]
```diff
@@ -112,22 +100,22 @@ You do not need to add a script for Helm 3. Harness includes Helm 3 support in a
 
 ### Pip
 
-Ensure that you run `apt-get update` before running any `apt-get` commands.
+Ensure that you run `microdnf update` before running any `microdnf` commands.
```
Comment (kat-enos) [inline docs/platform/2_Delegates/delegate-reference/common-delegate-profile-scripts.md:103]:
```suggestion
Run `microdnf update` before you run `microdnf` commands.
```

File: docs/platform/2_Delegates/advanced-installation/automate-delegate-installation.md [hunk 4]
```diff
@@ -65,18 +65,22 @@ The Delegate Environment Variables are described in the relevant Delegate instal
 
 ### Step 2: Rename the New Delegate
 
-The process you use to rename a delegate depends on its type. For Docker delegates, you change the name in one environment variable in the Docker compose file. For the Kubernetes delegate, you change multiple instances of the name.
+The process you use to rename a delegate depends on its type. For a Docker delegate, you change the name on the command line. For a Kubernetes delegate, you change multiple instances of the name.
```
Comment (kat-enos) [inline docs/platform/2_Delegates/advanced-installation/automate-delegate-installation.md:68]:
```suggestion
The process you use to rename a delegate depends on its type. For a Docker delegate,  you can change the name on the command line. For a Kubernetes delegate, you must change multiple instances of the name.
```

File: docs/platform/2_Delegates/customize-delegates/build-custom-delegate-images-with-third-party-tools.md [hunk 5]
```diff
@@ -31,7 +31,7 @@ You can build on either of the following Harness-provided images.
 | Harness Delegate Docker image | A publicly available Docker image providing Harness Delegate. |
 | Harness Minimal Delegate Docker image | A minimal delegate image available in Docker Hub at <https://hub.docker.com/r/harness/delegate/tags>. |
 
-You can use the `latest` version minimal image from the Docker repository.
+You can use the  last published `YY.MM.XXXXX` version of the minimal image from the Docker repository.
```
Comment (kat-enos) [inline docs/platform/2_Delegates/customize-delegates/build-custom-delegate-images-with-third-party-tools.md:34]:
```suggestion
Use the  last published `YY.MM.XXXXX` version of the minimal image from the Docker repository.
```

File: docs/platform/2_Delegates/delegate-reference/common-delegate-profile-scripts.md [hunk 6]
```diff
@@ -112,22 +100,22 @@ You do not need to add a script for Helm 3. Harness includes Helm 3 support in a
...
-apt-get -y install python-pip  
+microdnf -y install python3-pip  
 # Check pip install  
 pip -v
 ```
 
 ### Unzip
 
-Ensure that you run `apt-get update` before running any `apt-get` commands.
+Ensure that you run `microdnf update` before running any `microdnf` commands.
```
Comment (kat-enos) [inline docs/platform/2_Delegates/delegate-reference/common-delegate-profile-scripts.md:114]:
```suggestion
Run `microdnf update` before you run `microdnf` commands.
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegate-requirements-and-limitations.md [hunk 7]
```diff
@@ -30,11 +30,7 @@ One Delegate size does not fit all use cases, so Harness let's you pick from sev
 ![](./static/delegate-requirements-and-limitations-00
 Remember that the memory and CPU requirements are for the Delegate only. You Delegate host/pod/container will need more computing resources for its operations systems and other services such as Docker or Kubernetes.
 
-The Delegate runs on a Linux/UNIX server or container.
-
-Ensure that you provide the minimum memory for the Delegate and enough memory for the host/node system. For example, an AWS EC2 instance type such as m5a.xlarge has 16GB of RAM, 8 for the Delegate and 8 for the remaining operations.
-
-The ​Shell Script Delegate requires cURL 7.64.1 or later.
+The Delegate runs on a Linux/UNIX container.
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegate-requirements-and-limitations.md:33]:
```suggestion
The delegate runs in a Linux/UNIX container.
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegate-requirements-and-limitations.md [hunk 8]
```diff
@@ -30,11 +30,7 @@ One Delegate size does not fit all use cases, so Harness let's you pick from sev
 ![](./static/delegate-requirements-and-limitations-00
 Remember that the memory and CPU requirements are for the Delegate only. You Delegate host/pod/container will need more computing resources for its operations systems and other services such as Docker or Kubernetes.
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegate-requirements-and-limitations.md:31]:
```suggestion
The requirements for memory and CPU support only the delegate. The delegate host, pod, and container require additional compute resources to support other operations, systems, and services such as Docker or Kubernetes.
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegate-requirements-and-limitations.md [hunk 9]
```diff
@@ -76,7 +70,7 @@ For steps on adding certs or other software to the Delegate, see [Common Delegat
 
 ### Delegate Access Requirements
 
-* The Harness Delegate does NOT require root account access, but the Kubernetes and Docker Delegates run as root by default. If you do not need to install applications using Delegate Profiles, then you can use a non-root account or install the application without the Delegate.  
+* The Harness Delegate does NOT require root account access, but the Kubernetes and Docker Delegates run as root by default. If you do not need to install applications using INIT_SCRIPT, then you can use a non-root account or install the application without the Delegate.
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegate-requirements-and-limitations.md:73]:
```suggestion
* Delegates do not require root account access. Kubernetes and Docker delegates do, however, run as root by default. If you do not need to install applications during the initialization process (`INIT_SCRIPT`), you can use a non-root account or install the application without the delegate.  
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegate-requirements-and-limitations.md [hunk 10]
```diff
@@ -76,7 +70,7 @@ For steps on adding certs or other software to the Delegate, see [Common Delegat
 
 ### Delegate Access Requirements
 
-* The Harness Delegate does NOT require root account access, but the Kubernetes and Docker Delegates run as root by default. If you do not need to install applications using Delegate Profiles, then you can use a non-root account or install the application without the Delegate.  
+* The Harness Delegate does NOT require root account access, but the Kubernetes and Docker Delegates run as root by default. If you do not need to install applications using INIT_SCRIPT, then you can use a non-root account or install the application without the Delegate.  
 See [Non-Root Delegate Installation](/docs/platform/2_Delegates/advanced-installation/non-root-delegate-installation.md).
 * If you do not run the Delegate as root, be aware that you cannot install any software using a [Delegate Initialization Script](/docs/platform/2_Delegates/delegate-reference/common-delegate-profile-scripts.md).
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegate-requirements-and-limitations.md:75]:
```suggestion
* If you do not run the delegate as root, you cannot use [delegate initialization scripts](/docs/platform/Delegates/delegate-reference/common-delegate-profile-scripts.md) to install software.
```

File: docs/platform/2_Delegates/manage-delegates/select-delegates-with-selectors.md [hunk 11]
```diff
@@ -14,7 +14,16 @@ In some cases, you might want Harness to select specific delegates. In these cas
 
 ### Review: Delegate tags
 
-A delegate tag is added to your delegate automatically when you set it up in Harness. The tag is added using the name you give your Delegate.
+A delegate tag with the same name as your delegate is added to your delegate automatically when you set it up in Harness. You can also give additional comma separated tags in the helm command line or in kubernetes yaml file like below
```
Comment (kat-enos) [inline docs/platform/2_Delegates/manage-delegates/select-delegates-with-selectors.md:17]:
```suggestion
A delegate tag with the same name as your delegate is automatically added to your delegate during the configuration process. You can add one or more comma-separated tags on the `helm` command line or in the Kubernetes YAML file, as shown in the following example.
```

File: docs/platform/2_Delegates/manage-delegates/select-delegates-with-selectors.md [hunk 12]
```diff
@@ -14,7 +14,16 @@ In some cases, you might want Harness to select specific delegates. In these cas
...
+```
+...  
+    env:
+    ....
+    - name: DELEGATE_TAGS
+      value: "tag1,tag2"
+...
+```
 
 You can also add more tags in the **Tags** field during the setup process:
```
Comment (kat-enos) [inline docs/platform/2_Delegates/manage-delegates/select-delegates-with-selectors.md:28]:
```suggestion
You can also add tags to the **Tags** field during the setup process:
```

File: docs/platform/2_Delegates/secure-delegates/trust-store-override-for-delegates.md [hunk 13]
```diff
@@ -10,7 +10,7 @@ helpdocs_is_published: true
 
 Harness Delegates perform most Harness tasks. Delegates make outbound TLS/SSL connections to the Harness SaaS platform to obtain these task assignments. The TLS/SSL connection from the delegate to Harness requires a trusted certificate.
 
-Harness Delegate ships with a Java Runtime Environment (JRE) that includes a default trusted certificate in its [truststore](https://docs.oracle.com/cd/E19830-01/819-4712/ablqw/index.html) (located at `jdk8u242-b08-jre/lib/security/cacerts`). This truststore uses multiple trusted certificates, however, you might want to limit them to conform to your company's security protocols.
+Harness Delegate ships with a Java Runtime Environment (JRE) that includes a default trusted certificate in its [truststore](https://docs.oracle.com/cd/E19830-01/819-4712/ablqw/index.html) (located at `/etc/pki/java/cacerts`). This truststore uses multiple trusted certificates, however, you might want to limit them to conform to your company's security protocols.
```
Comment (kat-enos) [inline docs/platform/2_Delegates/secure-delegates/trust-store-override-for-delegates.md:13]:
```suggestion
Harness Delegate ships with a Java Runtime Environment (JRE) that includes a default trusted certificate in its [truststore](https://docs.oracle.com/cd/E19830-01/819-4712/ablqw/index.html) (located at `/etc/pki/java/cacerts`). This truststore uses multiple trusted certificates. You can limit the number you use based on your company security protocols.
```

File: docs/platform/2_Delegates/secure-delegates/trust-store-override-for-delegates.md [hunk 14]
```diff
@@ -10,7 +10,7 @@ helpdocs_is_published: true
...
 
 Harness' only requirement is that the JRE truststore includes the certificate delegates use to trust Harness (app.harness.io).
```
Comment (kat-enos) [inline docs/platform/2_Delegates/secure-delegates/trust-store-override-for-delegates.md:15]:
```suggestion
The JRE truststore must include the certificate that delegates require to establish trust with Harness (app.harness.io).
```

File: docs/platform/2_Delegates/secure-delegates/trust-store-override-for-delegates.md [hunk 15]
```diff
@@ -131,7 +131,7 @@ Edit the Kubernetes delegate YAML file. It's named **harness-delegate.yaml**.
 
 Open the delegate YAML file in a text editor.
 
-In the `StatefulSet` manifest, under `env`, locate `JAVA_OPTS`.
+In the `Deployment` manifest, under `env`, locate `JAVA_OPTS`.
```
Comment (kat-enos) [inline docs/platform/2_Delegates/secure-delegates/trust-store-override-for-delegates.md:134]:
```suggestion
In the `Deployment` specification, locate the `env` field, and then find the `JAVA_OPTS` environment variable.
```

File: docs/platform/2_Delegates/delegate-reference/common-delegate-profile-scripts.md [hunk 16]
```diff
@@ -23,18 +23,6 @@ You can add any commands supported on the host/container/pod running the delegat
 
 The base image for the delegate is Ubuntu 18.04 or later. This means you can use any default Ubuntu package in delegate script.
```
Comment (kat-enos) [inline docs/platform/2_Delegates/delegate-reference/common-delegate-profile-scripts.md:24]:
```suggestion
The base image for the delegate uses Ubuntu 18.04 or later. This means you can use any default Ubuntu package in delegate script.
```

File: docs/platform/2_Delegates/delegate-reference/common-delegate-profile-scripts.md [hunk 17]
```diff
@@ -21,19 +21,7 @@ For more information about running scripts, see [Install Software on the Delegat
 
 You can add any commands supported on the host/container/pod running the delegate. Linux shell commands are most common. If `kubectl`, Helm, or Docker is running on the host/container/pod where you install the delegate, then you can use their commands. Kubernetes and Docker delegates include Helm.
```
Comment (kat-enos) [inline docs/platform/2_Delegates/delegate-reference/common-delegate-profile-scripts.md:22]:
```suggestion
You can add any command that the host, container, or pod running the delegate supports. Linux shell commands are most common. If `kubectl`, Helm, or Docker is running on the host, container, or pod where you install the delegate, you can use those commands. Kubernetes and Docker delegates include Helm.
```

**Review by kat-enos — APPROVED** [review]
This merge includes the deletion of the following files. I think those files may show as deletions because of a directory change from "2_Delegates" to "Delegates". Please confirm that your intent was not to delete the following files:

- https://developer.harness.io/docs/platform/Delegates/customize-delegates/custom-delegate
- https://developer.harness.io/docs/platform/Delegates/customize-delegates/install-delegates-with-third-party-tools
- https://developer.harness.io/docs/platform/Delegates/advanced-installation/non-root-delegate-installation
- https://developer.harness.io/docs/platform/Delegates/configure-delegates/run-scripts-on-delegates
- https://developer.harness.io/docs/platform/Delegates/configure-delegates/support-for-delegate-docker-images

**Comment by raghuAtWings** [comment]
> @raghuAtWings , this merge includes the deletion of the following files. I think those files may show as deletions because of a path change from "2_Delegates" to "Delegates". Please confirm that your intent was not to delete the following files:
> 
> * https://developer.harness.io/docs/platform/Delegates/customize-delegates/custom-delegate
> * https://developer.harness.io/docs/platform/Delegates/customize-delegates/install-delegates-with-third-party-tools
> * https://developer.harness.io/docs/platform/Delegates/advanced-installation/non-root-delegate-installation
> * https://developer.harness.io/docs/platform/Delegates/configure-delegates/run-scripts-on-delegates
> * https://developer.harness.io/docs/platform/Delegates/configure-delegates/support-for-delegate-docker-images

@kat-enos confirming that we  should  delete them

**Review by kat-enos — APPROVED** [review]
Locally tested

**Pull request merged**